esp/src/src-react/components/Title.tsx

```diff
@@ -189,7 +189,7 @@ export const DevTitle: React.FunctionComponent<DevTitleProps> = ({
             if (currentUser.passwordIsExpired) {
                 alert(nlsHPCC.PasswordExpired);
                 setShowMyAccount(true);
-            } else if (currentUser.passwordDaysRemaining <= currentUser.passwordExpirationWarningDays) {
+            } else if (currentUser.passwordDaysRemaining && currentUser.passwordDaysRemaining <= currentUser.passwordExpirationWarningDays) {
```

Comment:
With htpasswd, seems like passwordDaysRemaining and passwordExpirationWarningDays will very likely both be null and satisfy that else if.